Neos.Fusion/Classes/Core/Runtime.php

```diff
@@ -907,25 +871,22 @@ protected function throwExceptionForUnrenderablePathIfNeeded($fusionPath, $fusio
     {
         if (isset($fusionConfiguration['__objectType'])) {
             $objectType = $fusionConfiguration['__objectType'];
-            throw new Exceptions\MissingFusionImplementationException(sprintf(
-                "The Fusion object `%s` cannot be rendered:
-					Most likely you mistyped the prototype name or did not define 
-					the Fusion prototype with `prototype(%s) < prototype ...` . 
-					Other possible reasons are a missing parent-prototype or 
-					a missing `@class` annotation for prototypes without parent.
-					It is also possible your Fusion file is not read because 
-					of a missing `include:` statement.",
-                $objectType,
-                $objectType
-            ), 1332493995);
+            throw new Exceptions\MissingFusionImplementationException(<<<MESSAGE
+                The Fusion object "$objectType" cannot be rendered:
+                Most likely you mistyped the prototype name or did not define
+                the Fusion prototype with "prototype($objectType) < prototype(...)".
+                Other possible reasons are a missing parent-prototype or
+                a missing "@class" annotation for prototypes without parent.
+                It is also possible your Fusion file is not read because
+                of a missing "include:" statement.
+                MESSAGE, 1332493995);
```

Comment:
I like my proposed Heredoc much better for multilines. Also note that there will be no unnecessary whitespace.

```
sprintf(
    "Fooobar
     bung buz"
)
```
will result in `"Fooobar\n    bung buz"`

but



```
<<<MESSAGE
    "Fooobar
     bung buz"
     MESSAGE
```
will result in `"Fooobar\nbung buz"`